virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -40,3 +43,40 @@ def create_disk(disk_type, path=None, size="500M", disk_format="raw", extra='',
     else:
         return libvirt.create_local_disk(disk_type, path=path, size=size,
                                          disk_format=disk_format, extra=extra)
+
+
+def create_primitive_disk(type_name, disk_device, device_target, device_bus,
```

Comment:
This signature doesn't indicate that only the xml is created as opposed to e.g. `create_disk` in the same file. Could you change the name please to `create_primitive_disk_xml`?

Why is this function "primitive"? I'm wondering if there's a way to name or document this function to justify its existence https://github.com/avocado-framework/avocado-vt/blob/7f273c05d787908b5f7cef415a6ef9fee9ea1a03/virttest/utils_test/libvirt.py#L1611 ?

Could the comment maybe indicate which use cases are covered by `create_primitive_disk` as opposed to `create_disk_xml`? Is this a `ceph` specific function as suggested by the parameter doc (`ceph_disk_name`)? Should this be reflected in its name?

Reply:
update name to create_primitive_disk_xml